refiner-io/mobile-sdk-react-native
# Upgrade versions and remove databinding

# What it does

1. This pull request upgrades Refiner Android SDK to version 1.3.5 and React-native version to 0.73.2
2. Removes databinding usage

# How to test it

1. Remove databinding build feature in app build.gradle
4. Run the app
5. Open app and see the survey shows up successfully

# Acceptance criteria

* No issues appears during project build
* Example app runs successfully
